recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
```

Comment:
IMHO, yes, for many reasons:

- download size is (probably) smaller
- tarballs + sha/md5 is a stricter definition of version than a git tag (tags can change to other revisions)